search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -75,12 +76,83 @@ function displayTopResultsForCurrentTrend(countryCode) {
         countryData[0].averageSentiment + '<br>';
 
     for (let i = 0; i < results.length; i++) {
-      resultElement.innerHTML += '<a href=' + results[i].link + '>' +
-        results[i].htmlTitle + '</a><br>' + results[i].snippet + '<br>'
-        + 'Sentiment Score: ' + results[i].score + '<br>';
+      let searchResultIndex = i + 1;
```

Comment:
What is the purpose for  increasing this by 1? Why don't you just have the for loop start from 1 and go until <= results.length?

Also why are you doing this?

Reply:
I think she's showing the index for each search result in the modal. But @ntarn can't you just use i+1 when you show it on the next line?

Reply:
I added a comment to show that I'm showing the index for each search result in the modal, and addressed Chenyu's concern.